pkg/bundle/sync_test.go

```diff
@@ -69,6 +78,16 @@ func Test_syncTarget(t *testing.T) {
 			expOwnerReference: true,
 			expNeedsUpdate:    true,
 		},
+		"if object doesn't exist with jks, expect update": {
+			object:            nil,
+			namespace:         corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: "test-namespace"}},
+			selector:          labelEverything,
+			withJks:           true,
+			expExists:         true,
+			expJks:            true,
+			expOwnerReference: true,
+			expNeedsUpdate:    true,
+		},
```

Comment:
suggestion: I don't know from these tests what would happen if I were to specify a target where the JKS key is (accidentally) the same as the key for the PEM trust store, i.e.:

```yaml
  target:
    configMap:
      key: "root-certs.pem"
    additionalFormats:
      jks:
        key: "root-certs.pem"
```

I'd expect we should error in this case. Could we handle it and add a test for it?

Reply:
I expect that we'd get an error from kubernetes for attempting to write an invalid configmap, but I agree - we should catch that ahead of time if we can.